andela/coding-geeks-ah-frontend
# #167484562 Add Search Functionality and Custom filtering based on parameters

### What does this PR do?
- Add Search Functionality and Custom filtering based on parameters.
### Description of Task to be completed?
- Create UI design for searching articles
- Test the functionality
### How should this be manually tested?
- Clone this branch.
- `npm start` to run the server
- On the navbar you'll see search icon and click on it.
- You'll go to the search page, from there you can type any word you want to search for and also you can select a filter option either by `title, author, or tag`
`N.B:` live search start to work if your search query has at least length of 3.
### What are the relevant pivotal tracker stories?
- [Search Functionality](https://www.pivotaltracker.com/story/show/167484562)
